provider/aws/go.mod

```diff
@@ -306,3 +251,7 @@ require (
 	mvdan.cc/lint v0.0.0-20170908181259-adc824a0674b // indirect
 	mvdan.cc/unparam v0.0.0-20220706161116-678bad134442 // indirect
 )
+
+replace github.com/nitrictech/nitric/core => ../../core
```

Comment:
if the cli imports the provider, this is not going to work.
basically you need to undo these replaces. One way is to have a PR that just adds core
then a second one that does the remainder of the changes.

Reply:
for reproducible builds this becomes a problem without replace as your sum file will always be behind and defeats the point of having a monorepo structure in the first place.

Will see if there is a way we can address this with go workspaces instead

Reply:
Should be able to get this to work with workspaces and remove the replace directives (wasn't working before). Will see if I can resync the dependencies first.

Reply:
replace directives removed and workspace resynced.